packages/api-db-mongodb/src/adapter.ts

```diff
@@ -98,22 +98,22 @@ export class MongoDBAdapter implements DBAdapter {
     this.db = db
 
     this.peer = new MongoDBPeerAdapter(db)
-    this.user = new MongoDBUserAdapter(db, bcryptHashCostFactor, locale)
-    this.userRole = new MongoDBUserRoleAdapter(db, locale)
-    this.subscription = new MongoDBSubscriptionAdapter(db, locale)
+    this.user = new MongoDBUserAdapter(db, bcryptHashCostFactor)
+    this.userRole = new MongoDBUserRoleAdapter(db)
+    this.subscription = new MongoDBSubscriptionAdapter(db)
     this.session = new MongoDBSessionAdapter(db, this.user, this.userRole, sessionTTL)
```

Comment:
Mutations follow with more PRs, this PR is basically just queries :D
Because for mutations I need to figure out how to do replica sets first with docker-compose